src/Compilers/CSharp/Portable/Symbols/Source/SourceClonedParameterSymbol.cs

```diff
@@ -56,6 +56,8 @@ internal override bool IsMetadataOptional
             }
         }
 
+        internal override BoundExpression ExplicitDefaultExpression => _originalParam.ExplicitDefaultExpression;
```

Comment:
ClonedParameters are used as proxies to existing parameters - like when an indexer needs to have similar parameters as in the property. 
You do not need/want to analyse the clones. This scenario should not arise for local functions anyways.